src/main/java/ilex/util/JavaLoggerAdapter.java

```diff
@@ -106,60 +114,69 @@ public static void initialize(ilex.util.Logger ilexLogger, boolean _doForward, S
 		if (initialized)
 			return;
 		initialized = true;
-		doForward = _doForward;
-		JavaLoggerAdapter.ilexLogger = ilexLogger;
-		
-		java.util.logging.Logger globalLogger = LogManager.getLogManager().getLogger(globalName);
-		Handler handlers[] = globalLogger.getHandlers();
-		while (handlers != null && handlers.length > 0)
-		{
-			//&& handlers[0] instanceof ConsoleHandler)
-			globalLogger.removeHandler(handlers[0]);
-			handlers = globalLogger.getHandlers();
-		}
-		globalLogger.addHandler(instance());
-		globalLogger.setLevel(Level.ALL);
-		
-		for(String path : paths)
+
+		/* if the property wasn't set. tweak the log, otherwise let the user have what they wanted. */
+		String existingLoggingConfig = System.getProperty("java.util.logging.config.file");
+		if (existingLoggingConfig == null)
 		{
-			java.util.logging.Logger logger = java.util.logging.Logger.getLogger(path);
-			if (logger == null)
-				Logger.instance().debug3("logger is null for path '" + path + "'.");
-			else if (logger == globalLogger)
-				continue;
-			else
+			OpenDcsJulFormatter formatter = new OpenDcsJulFormatter();
+			java.util.logging.Logger globalLogger = java.util.logging.Logger.getLogger("");
+			Handler handlers[] = globalLogger.getHandlers();
+			if (handlers != null && handlers.length > 0)
 			{
-				handlers = logger.getHandlers();
-				if (handlers == null || handlers.length == 0)
-					Logger.instance().debug3("Path '" + path + "' No loggers present.");
-				while (handlers != null && handlers.length > 0)
+				for( Handler h: handlers)
 				{
-					//&& handlers[0] instanceof ConsoleHandler)
-					Logger.instance().debug3("Path '" + path + "': Removing " + handlers[0].getClass().getName());
-					logger.removeHandler(handlers[0]);
-					handlers = logger.getHandlers();
+					if(h instanceof ConsoleHandler) {
+						h.setFormatter(formatter);
+						h.setLevel(Level.SEVERE);
+					}
 				}
-				logger.addHandler(instance());
-				logger.setLevel(Level.ALL);
 			}
+
+			for(String path : HIDE_BY_DEFAULT)
+			{
+				setSevere(path, globalLogger);				
+			}
+
+			boolean trace = Boolean.parseBoolean(System.getProperty(CwmsConnectionPool.POOL_TRACE_PROPERTY,"false"));
```

Comment:
It feels odd to be referencing something with the name CWMS (CwmsConnectionPool) in a utility area of the code.   I would expect Utility code to avoid that type of dependency.

Reply:
Yes, the property started within the CwmsConnectionPool and for this purpose I just needed a way to get it and see the results. Long term, as said, in a comment below the "trace connections" property will be moved somewhere generic.